### ISSUE TITLE
Can't search for emails that contain a plus sign in Dashboard order search

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2867]
### Expected behavior
Should be able to search for any email including ones that contain a plus sign

### Actual Behavior
Emails that contain a plus sign (like testing emails) do not show up in search

### Steps to Reproduce the Behavior
1. Create an order with an email with a plus sign in it
1. Go to the new order search in the dashboard
1. Search for this email
1. Observe that it doesn't show up

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.2-ce
```

**Comment by aaronjudd** [comment]
Also verify this isn't a script security issue, escaping regex, etc..

**Comment by zenweasel** [comment]
To be clear we "clean" phone numbers for search so that (323), 323- and 323 all match when searching for 323 so this means that if you search for (323) it's never going to find a match. But we display the original phone # however is was input. 

So the ph# part of this is probably invalid.

**Comment by zenweasel** [comment]
Closed via #2926
